openff/benchmark/tests/test_validate_and_assign_ids.py

```diff
@@ -0,0 +1,400 @@
+from openforcefield.utils.toolkits import GLOBAL_TOOLKIT_REGISTRY, OpenEyeToolkitWrapper, RDKitToolkitWrapper
+from openff.benchmark.utils.validate_and_assign_ids import validate_and_assign
+from openff.benchmark.utils.utils import get_data_file_path, temporary_cd
```

Comment:
I think I recall that you wanted to switch to using `temporary_cd` once you were satisfied things were working. I'd recommend using [`tmpdir` from `pytest`](https://docs.pytest.org/en/stable/tmpdir.html#the-tmpdir-fixture); this will create directories in e.g. `/tmp` and rotate them between runs, keeping some fixed number around for inspection.